package.json

```diff
@@ -27,7 +27,7 @@
     "suitest": "bin/suitest"
   },
   "scripts": {
-    "test": "mocha --no-timeouts",
+    "test": "mocha",
```

Comment:
Was it the reason for UTs being unstable? How come?

Reply:
there was for developming only. I added it when I tried make success mock for package-json library http request